ssh_check/assets/configuration/spec.yaml

```
@@ -63,4 +63,15 @@ files:
               example: false
               type: boolean
 
+          - name: force_sha1
+            description: |
+              Some older ssh servers only support SHA1 encryption. Moreover some don't have the `server-sig-algs`
+              extension installed which means they don't tell their clients that they only support SHA1.
+              The client we use assumes SHA2 encryption, so it crashes when connecting to such a server.
+              While it's strongly recommended to upgrade the ssh server to support the more secure SHA2 we can force
+              the client to use SHA1 encryption by setting this option to 'true'.
```

```suggestion
              The client we use assumes SHA2 encryption, so it crashes when connecting to such a server.
              While it's strongly recommended to upgrade the SSH server to support the more secure SHA2, we can force
              the client to use SHA1 encryption by setting this option to 'true'.
```
If possible, please reword these sentences to not use "we." Consider replacing it with "Datadog," "the system," or the specific name of the software system